.github/workflows/node.js.yml

```diff
@@ -12,12 +12,18 @@ on:
 jobs:
   browser:
     runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x, 16.x]
+        ts-project: [src/tsconfig.json, src/tsconfig-es6.json]
+
     steps:
       - uses: actions/checkout@v2
       - uses: nuget/setup-nuget@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: ${{ matrix.node-version }}
```

Comment:
Why should we test every NodeJS version on windows? Just curious about it

Reply:
Good question. The purpose of this job is to test in the browser.